examples/color/named_colors.py

```diff
@@ -59,15 +60,16 @@ def plot_colortable(colors, title, sort_colors=True, emptycols=0):
         y = row * cell_height
 
         swatch_start_x = cell_width * col
-        swatch_end_x = cell_width * col + swatch_width
         text_pos_x = cell_width * col + swatch_width + 7
 
         ax.text(text_pos_x, y, name, fontsize=14,
                 horizontalalignment='left',
                 verticalalignment='center')
 
-        ax.hlines(y, swatch_start_x, swatch_end_x,
-                  color=colors[name], linewidth=18)
+        ax.add_patch(
+            Rectangle(xy=(swatch_start_x, y-9), width=swatch_width,
+                      height=18, fill=True, facecolor=colors[name])
```

Comment:
```suggestion
                      height=18, facecolor=colors[name])
```
Filling is default for patches, no need to specify this explicitly.